includes/api/class-wc-admin-rest-products-controller.php

```diff
@@ -78,6 +84,9 @@ protected function prepare_objects_query( $request ) {
 			$args['search'] = $request['search'];
```

Comment:
Should there be a `trim()` here? _(related to a comment below)_

Reply:
It's already trimmed elsewhere, but we can trim here instead and remove the other trims to make this more manageable.